pkgs/development/python-modules/beanhub-cli/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pytz
+, pydantic
+, pydantic-core
+, pydantic-settings
+, pyyaml
+, jinja2
+, rich
+, fastapi
+, click
+, uvicorn
+, starlette-wtf
+, beancount-parser
+, beancount-black
+, beanhub-import
+, beanhub-forms
+, poetry-core
+, httpx
+, pytestCheckHook
```

Comment:
I'd order the python modules ASCII-betically, and keep `pytestCheckHook` with the other packaging stuff.

Reply:
```suggestion
, pytestCheckHook
, beancount-black
, beancount-parser
, beanhub-forms
, beanhub-import
, click
, fastapi
, httpx
, jinja2
, poetry-core
, pydantic
, pydantic-core
, pydantic-settings
, pytz
, pyyaml
, rich
, starlette-wtf
, uvicorn
```

Reply:
@ambroisie hey, thanks! sorry I am pretty new to contributing to nixpkgs. I usually run

```
nix fmt .
```

for my own nix projects. I just tried to run it but it says

```
error: flake 'git+file:///home/fangpen/workspace/playground/nixpkgs' does not provide attribute 'formatter.x86_64-linux'
```

ideally I think there should be a tool for this. but if not it's okay, thanks for calling it out anyway 🙇‍♀️